pkg/systemd/services/services.jsx

```diff
@@ -439,6 +442,14 @@ class ServicesPageBody extends React.Component {
                                     }, ex => this.setState({ error: cockpit.format(_("Listing of unit failed: $0"), ex.toString()), loadingUnits: false })));
                                 });
 
+                                let counter = 0;
+                                this.setState({ currentStatus: cockpit.format(_("Loading unit $0/$1"), counter, promisesLoad.length) });
+                                for (const p of promisesLoad) {
+                                    counter++;
+                                    p.then(() => this.setState({ currentStatus: cockpit.format(_("Loading unit $0/$1"), counter, promisesLoad.length) }))
```

Comment:
The services page already does too much on startup (we really shouldn't be doing hundreds of D-Bus calls!), and now this does hundreds of gettext calls and state updates in addition. This should at least use `throttle-debounce` or some simple `if (counter % 10 == 0)` or similar.

Reply:
ok, I can drop this. It was really just me playing around seeing if it gets stuck on some specific unit